adding_species_manuscript.md

```diff
@@ -389,31 +391,238 @@ Table of species/demographic scenarios added/worked on by community since origin
+    year=2006,
+    author="Pombi et al.",
+    reasons={stdpopsim.CiteReason.REC_RATE},
+)
+```
+
+To verify if we have progressed on the tests units, we can type again:
+
+```
+python -m pytest tests/test_AnoGam.py
+```
+### 2) Mutation rates
+
+Editing the species.py file concerning the mutation rate is very similar to the recombination rate. It turns out that there is no great estimates for mutation rate in Anopheles but in a recent population genomics effort by the Ag1000G consortium it has relied on Drosophila estimates from Schrider et al. (2013) that set u=5.49e-9. We'll go with that:
+
+```
+_overall_rate = 5.49e-9
+_mutation_rate = {
+    "2L": _overall_rate,
+    "2R": _overall_rate,
+    "3L": _overall_rate,
+    "3R": _overall_rate,
+    "X": _overall_rate,
+    "Mt": _overall_rate
+}
+```
+We should again add a citation for the mutation rate and then edit the _mutation_rate dict that was templated for us:
+
+```
+_Ag1000G = stdpopsim.Citation(
+    doi="https://doi.org/10.1038/nature24995",
+    year=2017,
+    author="Ag1000G Consortium",
+    reasons={stdpopsim.CiteReason.MUT_RATE, stdpopsim.CiteReason.GEN_TIME, stdpopsim.CiteReason.POP_SIZE},
+)
+```
+
+### 3) Assembly
+We need to add a citation for the assembly:
+Ensembl points me to this publication for their current assembly by Sharakhova et al. (2006):
+```
+_SharakhovaEtAl = stdpopsim.Citation(
+    doi="https://doi.org/10.1186/gb-2007-8-1-r5",
+    year=2006,
+    author="Sharakhova et al.",
+    reasons={stdpopsim.CiteReason.ASSEMBLY},
+)
+```
+and all this information is put it together to create the genome object as:
+
+```
+_genome = stdpopsim.Genome.from_data(
+    genome_data.data,
+    recombination_rate=_recombination_rate,
+    mutation_rate=_mutation_rate,
+    citations=[
+        _SharakhovaEtAl,
+        _Ag1000G,
+        _PombiEtAl,
+    ],
+)
+```
+
+### 4) Population size and generation time
+
+Finally we can edit the species definition in species.py to include a default population size and generation time:
+
+```
+_species = stdpopsim.Species(
+    id="AnoGam",
+    ensembl_id="anopheles_gambiae",
+    name="Anopheles gambiae",
+    common_name="Anopheles gambiae",
+    genome=_genome,
+    generation_time=1/11,
+    population_size=6e6, #Ghana population
+    citations=[_Ag1000G]
+)
+```
+
+To test if it works we can type at the command line:
+
+```
+python -m stdpopsim AnoGam --help
+```
+
+If successfully set, the command above returns a description of the species with details of the species parameters.
+
+To ensure that all is set it is required that all unit tests pass, so test again:
+
+```
+python -m pytest tests/test_AnoGam.py
+```
+
+If all pass, then it is possible to start a pull request for this species
+
+### Initiating a Pull Request for your species
+Once all the information for the species is included and all unit tests passes, then next step is to commit all changes the online repository branch and start a pull request (PR) on Github. 
```

Comment:
"... and all unit tests pass, the next step is to commit all changes to the online ..."

Reply:
done